src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -3234,6 +3238,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)Windows_UI_Xaml_Input\CopyToClipboard\CopyToClipboard.xaml.cs">
       <DependentUpon>CopyToClipboard.xaml</DependentUpon>
     </Compile>
+    <Compile Include="$(MSBuildThisFileDirectory)Windows_UI_Xaml_Input\Keyboard\Keyboard_Clickthrough_Modal.xaml.cs">
+      <DependentUpon>$fileinputname$.xaml</DependentUpon>
```

Comment:
There's a defect in VSWIN currently that cuases this. You'll need to change this to be the XAML filename. See https://developercommunity.visualstudio.com/content/problem/668062/shared-project-invalid-data-fileinputnamexaml.html.

```suggestion
      <DependentUpon>Keyboard_Clickthrough_Modal.xaml</DependentUpon>
```